src/Symfony/Component/Finder/Iterator/MultiplePcreFilterIterator.php

```diff
@@ -101,6 +101,25 @@ protected function isRegex($str)
         return false;
     }
 
+    protected function hasRegexModifiers(string $str, array $modifiers)
```

Comment:
I'm not sure we need this logic: let's just add "i" and not care about duplicates, that'd behave the same in the end, isn't it?

Reply:
That's right, duplicates behave the same.